plugins/redux/extend.js

```diff
@@ -0,0 +1,17 @@
+const base = {
+  _app: {
+    import: [
+      "import React from 'react'",
+      "import { Provider } from 'react-redux'",
+      "import store from '../src/redux/store'",
```

Comment:
```suggestion
      "import React from 'react'",
      "import { Provider } from 'react-redux'",
      "",
      "import store from '../src/redux/store'",
```
araya bir boşluk atabiliriz